src/pages/Hints.tsx

```diff
@@ -1,36 +1,33 @@
-import { useState } from "react"
+import { useState } from "react";
 
-type hints = {userId: 1|2,hint:string,isShow:boolean}[]
-type life = 0 | 1 | 2 | 3 | 4 | 5
+type hints = { userId: 1 | 2; hint: string; isShow: boolean }[];
+type life = 0 | 1 | 2 | 3 | 4 | 5;
 
-const mockhints:hints = [
-    {userId:1,hint:"ทดสอบ",isShow:true},
-    {userId:2,hint:"ทดสอบ",isShow:true},
-    {userId:1,hint:"ทดสอบ",isShow:true},
-    {userId:2,hint:"ทดสอบ",isShow:true},
-    {userId:1,hint:"ทดสอบ",isShow:true},
-    {userId:2,hint:"ทดสอบ",isShow:true},
-    {userId:1,hint:"ทดสอบ",isShow:true},
-    {userId:2,hint:"ทดสอบ",isShow:true},
-    {userId:1,hint:"ทดสอบ",isShow:true},
-    {userId:2,hint:"ทดสอบ",isShow:true},
-    {userId:1,hint:"ทดสอบ",isShow:false},
-    {userId:2,hint:"ทดสอบ",isShow:false},
-    {userId:1,hint:"ทดสอบ",isShow:false},
-    {userId:2,hint:"ทดสอบ",isShow:false},
-    {userId:1,hint:"ทดสอบ",isShow:false},
-    {userId:2,hint:"ทดสอบ",isShow:false},
-    {userId:1,hint:"ทดสอบ",isShow:false},
-    {userId:1,hint:"ทดสอบ",isShow:false},
-
-]
+const mockhints: hints = [
+  { userId: 1, hint: "ทดสอบ", isShow: true },
+  { userId: 2, hint: "ทดสอบ", isShow: true },
+  { userId: 1, hint: "ทดสอบ", isShow: true },
+  { userId: 2, hint: "ทดสอบ", isShow: true },
+  { userId: 1, hint: "ทดสอบ", isShow: true },
+  { userId: 2, hint: "ทดสอบ", isShow: true },
+  { userId: 1, hint: "ทดสอบ", isShow: true },
+  { userId: 2, hint: "ทดสอบ", isShow: true },
+  { userId: 1, hint: "ทดสอบ", isShow: true },
+  { userId: 2, hint: "ทดสอบ", isShow: true },
+  { userId: 1, hint: "ทดสอบ", isShow: false },
+  { userId: 2, hint: "ทดสอบ", isShow: false },
+  { userId: 1, hint: "ทดสอบ", isShow: false },
+  { userId: 2, hint: "ทดสอบ", isShow: false },
+  { userId: 1, hint: "ทดสอบ", isShow: false },
+  { userId: 2, hint: "ทดสอบ", isShow: false },
+  { userId: 1, hint: "ทดสอบ", isShow: false },
+  { userId: 1, hint: "ทดสอบ", isShow: false },
+];
 
 const Hints = () => {
-    const [hints] = useState<hints>(mockhints)
-    const [life] = useState<life>(0)
-  return (
-    <div>Hints</div>
-  )
-}
+  const [hints] = useState<hints>(mockhints);
+  const [life] = useState<life>(0);
+  return <div>Hints</div>;
+};
 
-export default Hints
\ No newline at end of file
+export default Hints;
```

Comment:
this file is not used?

Reply:
It not include in this section ka🥺 I code in wrong file;-;